index.html

```diff
@@ -0,0 +1,359 @@
+        <div class="row">
+          <div class="col-lg-7 col-xl-6 col-sm-12 firstSpain_block">
+            <div class="row">
+              <div class="col-12"><p>Spain</p></div>
+              <div class="col-12"><h3>Grand Spain Madrid</h3></div>
+              <div class="col-12">
+                <p class="firstSpain_p">
+                  But I must explain to you how all this mistaken idea of
+                  account to denouncing pleasure and praising pain was account
+                  born and I account will give you a complete account of the
+                  system
+                </p>
+              </div>
+              <div class="col-12 firstSpain_button">
+                <a href="#" class="btn-1">book now</a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="secondSpain">
+      <div class="container">
+        <div class="row">
+          <div class="col-6"></div>
+          <div class="col-lg-7 col-xl-6 col-sm-12 secondSpain_block">
+            <div class="row">
+              <div class="col-12"><p>Spain</p></div>
+              <div class="col-12"><h3>Grand Spain Madrid</h3></div>
+              <div class="col-12">
+                <p class="secondSpain_p">
+                  But I must explain to you how all this mistaken idea of
+                  account to denouncing pleasure and praising pain was account
+                  born and I account will give you a complete account of the
+                  system
+                </p>
+              </div>
+              <div class="col-12 secondSpain_button">
+                <a href="#" class="btn-1">book now</a>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section class="tourists">
+      <div class="container">
+        <div class="row">
+          <div class="col-12">
+            <h2>What Our Tourists Say</h2>
+          </div>
+        </div>
+        <div class="row">
+          <div class="col-lg-4 col-md-12">
+            <img src="img/ReviewPhoto.png" alt="tourists" />
+          </div>
+          <div class="col-lg-8 col-md-12">
+            <div class="row wrapper">
+              <div class="col-12">
+                <h4>Sam Sumon</h4>
+              </div>
+              <div class="col-12">
+                <p>
+                  <span class="icon-quote-left"></span>
+                  But I must explain to you how all this mistaken idea of
+                  account to denouncinpleasure and praising pain But I must
+                  explain to you how all this mistaken idea of account to nai
+                  denouncing pleasure and praising pain was account born and I
+                  account will give you a complete account of the system was
+                  account born and I account will give you a nai the complete
+                  account of the system
+                </p>
```

Comment:
1. Использовать тег **blockquote**
2. Кавычки лучше сделать через ::before
```suggestion
                <blockquote>
                  But I must explain to you how all this mistaken idea of
                  account to denouncinpleasure and praising pain But I must
                  explain to you how all this mistaken idea of account to nai
                  denouncing pleasure and praising pain was account born and I
                  account will give you a complete account of the system was
                  account born and I account will give you a nai the complete
                  account of the system
                </blockquote>
```